src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -45,7 +45,8 @@
   <ItemGroup>
     <PackageReference Include="Desktop.Analyzers" Version="1.1.0">
       <PrivateAssets>All</PrivateAssets>
-    </PackageReference>    
+    </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.2" />    
```

Comment:
Please no external references. Anything that is not a framework (or DiagnosticsSource which is an exception) not allowed. If you want this reference very badly - I'd suggest moving this logic into `WindowsServer` package that already have Azure-specific telemetry initializers. And it will be logical to have this there as well.

I'd say in general it's a good idea to put it into `WindowsServer` package.

Reply:
Agreed - this was a slip-up and was never intended to be part of my changes (was added during some preliminary experimentation only). Removed #Closed